src/test/java/org/openrewrite/java/migrate/javax/RemoveEmbeddableIdTest.java

```diff
@@ -160,4 +160,50 @@ public class EmbeddableObject {
           )
         );
     }
+
+    @Test
+    void removeEmbeddableIdFromSubClass() {
+        //language=java
+        rewriteRun(
+          java(
+            """
+              import javax.persistence.EmbeddedId;
+              import javax.persistence.Entity;
+
+              @Entity
+              public class MainEntity {
```

Comment:
I should have been clearer about describing the sub-class test case. I was looking for something like this:
```
              import javax.persistence.Embeddable;
              import javax.persistence.Id;
              public class SomeClass {
                        @Embeddable
                        public class EmbeddableObject {
                            @Id
                            private int field;
                        }
              }
```

Reply:
If `SomeClass` also had it's own `@Id` annotated field, should that be left unchanged by this recipe?

Reply:
Conversely, if the outer `SomeClass` is `@Embeddable`, has an `@Id` field, and has an inner class without `@Embeddable` and an inner field with `@Id`, should that inner field annotation be removed as well?